docs/site/Fields-filter.md

```diff
@@ -50,14 +47,18 @@ Return customer information only with `name` and `address`, and hide their `id`:
 await customerRepository.find({fields: {id: false, name: true, address: true}});
 ```
 
+Or use stringified JSON format:
```

Comment:
Nitpick:
Other filters have the stringified JSON format example after the REST examples.